### DISCUSSION TITLE
Utenti riceventi creati dopo una segnalazione non riescono a vederla

### DISCUSSION THREAD

**Discussion by stefanolmo** [globaleaks/globaleaks-whistleblowing-software#3697] [General]
Utenti riceventi creati successivamente all'invio di una segnalazione non possono vedere la segnalazione stessa, pertanto, qualora l'utente ricevente presente a sistema fin dall'origine sia sostituito da un nuovo utente, quest'ultimo non potrà vedere lo storico delle segnalazioni.
C'è modo di cambiare questo comportamento al fine di mostrare vecchie segnalazioni a nuovi utenti riceventi?

**Comment by evilaliv3** [comment]
Le segnalazioni a sistema sono cifrate per gli utenti definiti a sistema.

Come tale se volete aggiungere degli utenti e dargli visibilità di segnalazioni esistenti dovete:
- come amministratore assegnare agli utenti pre-esistenti la possibilità di dare visibilità delle proprie segnalazioni a nuovi utenti
- come utente pre-esistente dovete selezionare le segnalazioni esistenti e concedere l'accesso ai nuovi utenti

**Reply by stefanolmo** [reply]
Grazie mille! Funziona benissimo

**Reply by stefanolmo** [reply]
PS: non funziona il pulsante per fare una donazione sul sito Globaleaks
